indexer/indexer_tests/editable_map_object_test.cpp

```diff
@@ -191,37 +191,52 @@ UNIT_TEST(EditableMapObject_GetNamesDataSource)
 
   TEST_EQUAL(namesDataSource.names.size(), 9, ("All names except the default should be pushed into "
                                            "data source plus empty mandatory names"));
-  TEST_EQUAL(namesDataSource.mandatoryNamesCount, 3,
+  TEST_EQUAL(namesDataSource.mandatoryNamesCount, 4,
              ("Mandatory names count should be equal == Mwm languages + user`s language"));
```

Comment:
Either "should be == x + y" or "should be equal ... + ...". The second variant is better.